.github/workflows/build-images.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Build Docker images for all OpenSlides services
+on: push
+env:
+  IMAGE_VERSION: 3.4.0-dev
+jobs:
+  build:
+    name: Builds Docker images
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Build images
+      run: docker/build.sh -D ghcr.io/${{ github.repository_owner }} -t $IMAGE_VERSION all
+
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" |
+           docker login ghcr.io --username ${{ github.actor }} --password-stdin
+
+    - name: Push images
+      run: |
+        IMAGE_ID=${{ github.repository_owner }}/openslides-server
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-repmgr
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-autoupdate
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-postfix
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-client
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
+
+        IMAGE_ID=${{ github.repository_owner }}/openslides-pgbouncer
+        IMAGE_ID=$(echo $IMAGE_ID | tr '[A-Z]' '[a-z]')
+        docker push ghcr.io/$IMAGE_ID:$IMAGE_VERSION
```

Comment:
This could be done by the docker-compose repo. But I'm ok if we are doing it here, so the tags are the same, even if it the same image.